CONTRIBUTING.md

```diff
@@ -2,12 +2,28 @@
 
 ## Sign the CLA
 
-Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.  Please see https://git.k8s.io/community/CLA.md for more info
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.
+  
+Please see https://git.k8s.io/community/CLA.md for more info
 
-### Contributing A Patch
+## Contributing steps
 
 1. Submit an issue describing your proposed change to the repo in question.
 1. The [repo owners](OWNERS) will respond to your issue promptly.
 1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
 1. Fork the desired repo, develop and test your code changes.
 1. Submit a pull request.
+
+## Test locally
+
+1. Setup tools
+    ```bash
+    $ go get -u github.com/golang/dep/cmd/dep
```

Comment:
fetch dep from master is not recommended.  They occaisonally break things on master.  Generally, you should use a dep released versoin.